kubeshop/botkube
# ADD recommended flag to plugin index

## Description

Changes proposed in this pull request:

- Add recommended flag to plugin index

## Testing


- run plugin index locally
- check if server returns recommended flag (for kubectl & k8s should be true)

## Related issue(s)


https://github.com/kubeshop/botkube-cloud/issues/732
